main.js

```diff
@@ -138,18 +138,100 @@ if (args.reqJs)
         console.log(edge.toString());
     });
 
-if (args.cg) {
-    function pp(v) {
-        if (v.type === 'CalleeVertex')
-            return astutil.ppPos(v.call);
-        if (v.type === 'FuncVertex')
-            return astutil.ppPos(v.func);
-        if (v.type === 'NativeVertex')
-            return v.name;
-        throw new Error("strange vertex: " + v);
+if (args.cgPath) {
+    function constructNodeFromCallVertex(callVertex) {
+        if (!callVertex.call.attr.hasOwnProperty('enclosingFunction')) {
+            return {
+                'function_name': 'toplevel',
+                "file_name": callVertex.call.attr.enclosingFile,
+                'function_position': callVertex.call.loc.start.line + ":" + callVertex.call.loc.start.column,
```

Comment:
If you would prefer one liner, you can write it as \`${callVertex.call.loc.start.line}:${callVertex.call.loc.start.column}\`

Reply:
o yeah, interpolation syntax, forgot about it. Thanks!